cogs/tools.py

```diff
@@ -134,6 +138,62 @@ async def ask_question(self, ctx, conversation_mode="true", *, question: str):
 
         await ctx.send(embed=embed)
 
+    @commands.command()
+    async def charinfo(self, ctx, *, characters: str):
+        """Shows you information about a number of characters. Only up to 25 characters at a time."""
```

Comment:
```suggestion
        """Shows you information about up to 25 characters."""
```
A bit more compact docstring